lib/auth-user.js

```diff
@@ -1,7 +1,19 @@
+const getUserDetails = require('./get-user-details');
+
 module.exports = function(req, res, next) {
   // TODO: user req.query.access_token to validate the user againts the osm api
```

Comment:
I'd add a `if (process.env.NODE_ENV==='test') { req.user = req.query.oauth_token; return next(); }` check or something so that we don't hit the osm api in tests.

Reply:
We should have one test that confirms the integration works, but most tests don't need to do the round trip.